src/Version/Version.cpp

```diff
@@ -67,6 +75,33 @@ std::string getOnnxMlirFullRepositoryVersion(bool toIncludeLLVM) {
   return buf;
 }
 
+std::string getProductFullVersion() {
+  std::string buf;
+  llvm::raw_string_ostream os(buf);
+#if defined(ONNX_MLIR_VENDOR)
+  os << ONNX_MLIR_VENDOR << " " << PRODUCT_VERSION_MAJOR << '.';
+  os << PRODUCT_VERSION_MINOR << '.';
+  os << PRODUCT_VERSION_PATCH << '-' << PRODUCT_ID;
+#endif
+  return buf;
+}
+
+std::string getOnnxMlirCommitVersion() {
+  std::string buf;
+  llvm::raw_string_ostream os(buf);
+  std::string OnnxMlirPath = getOnnxMlirRepositoryPath();
+  std::string OnnxMlirCommit = getOnnxMlirCommit();
```

Comment:
In the `#if defined(ONNX_MLIR_PRODUCT_VERSION)` path, `OnnxMlirCommit` is never used. Any reason not to move it to be inside of the `#else` and save a call?

Reply:
Updated!